service/request.go

```diff
@@ -203,7 +204,13 @@ func (w *ResponseWriter) ZioWriter() zio.WriteCloser {
 	if w.zw == nil {
 		w.Header().Set("Content-Type", api.FormatToMediaType(w.Format))
 		var err error
-		w.zw, err = anyio.NewWriter(zio.NopCloser(w), anyio.WriterOpts{Format: w.Format})
+		w.zw, err = anyio.NewWriter(zio.NopCloser(w), anyio.WriterOpts{
+			Format: w.Format,
+			ZNG: zngio.WriterOpts{
```

Comment:
Am wondering if this should be a pointer and if nil just does zngio.NewWriter?

Reply:
Excellent idea. I created #3978 for that.